src/Schema/Directives/LikeDirective.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+
+class LikeDirective extends BaseDirective implements ArgBuilderDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'GRAPHQL'
+enum LikePercentageLocation {
```

Comment:
That cannot be part of this definition, it must only contain the directive definition.

Reply:
Fixed by https://github.com/nuwave/lighthouse/pull/1649